### PR TITLE
Appveyor & Codecov setup for v3

### PR DESCRIPTION
**Pull request by janmeier**

### CODE REVIEW

**Comment by felixfbecker** [comment]
What about coverage? We should switch to codecov for v3 too

**Comment by felixfbecker** [comment]
lol at the branch name

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Sequelize
 
 [![Build Status](https://travis-ci.org/sequelize/sequelize.svg?branch=master)](https://travis-ci.org/sequelize/sequelize) [![Dependency Status](https://david-dm.org/sequelize/sequelize.svg)](https://david-dm.org/sequelize/sequelize) [![Test Coverage](https://codeclimate.com/github/sequelize/sequelize/badges/coverage.svg)](https://codeclimate.com/github/sequelize/sequelize)
+[![Windows Build status](https://ci.appveyor.com/api/projects/status/8xhttm9pxmbmtbwb/branch/master?svg=true)](https://ci.appveyor.com/project/felixfbecker/sequelize/branch/master)
```
Comment (felixfbecker) [inline README.md]: `master` must be `v3` here

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Sequelize
 
 [![Build Status](https://travis-ci.org/sequelize/sequelize.svg?branch=master)](https://travis-ci.org/sequelize/sequelize) [![Dependency Status](https://david-dm.org/sequelize/sequelize.svg)](https://david-dm.org/sequelize/sequelize) [![Test Coverage](https://codeclimate.com/github/sequelize/sequelize/badges/coverage.svg)](https://codeclimate.com/github/sequelize/sequelize)
```
Comment (felixfbecker) [inline README.md]: Need to remove codeclimate badge here

**Comment by felixfbecker** [comment]
🍒 -picked the MSSQL test fixes and Babel removal (caused errors because PS outputs a BOM and Babel choked on that while parsing package.json for config)

**Comment by felixfbecker** [comment]
@janmeier got tests passing and coverage report working. Ready to merge
@sdepold once this is merged you can kill the MSSQL server 🔫

**Comment by sdepold** [comment]
Nice!!

**Comment by janmeier** [comment]
Thanks for the help Felix! :-)

**Comment by BridgeAR** [comment]
I think it's great that Appveyor is used but is there actually still a possibility to test mssql on localhost?

**Comment by felixfbecker** [comment]
@BridgeAR Sure, with a Windows machine you can install MSSQL (which is free) and test locally. If you do MSSQL development you are likely to use Windows anyway. Connecting to a remote server was never really `localhost` in the first place :D

The EC2 instance is gone for the good, it was dog-slow and open to anyone on the internet. If you do not have a windows machine, you can open a PR and let AppVeyor run the tests.

**Comment by BridgeAR** [comment]
@felixfbecker I do not use Windows at all and I do not plan on doing so in the future ;-) But if I'm opening a PR I'd like to make sure everything is running. Currently I have a open PR here that has an issue with mssql for example (where I did not find a fix for) and I could not even try to fix that without mssql tests running on my machine.

**Comment by mickhansen** [comment]
SQL Server for Linux should be landing soon^tm

**Comment by janmeier** [comment]
> Mid-2017

https://blogs.microsoft.com/blog/2016/03/07/announcing-sql-server-on-linux/

**Pull request merged**